packages/common/src/hooks/usePremiumContent.ts

```diff
@@ -28,20 +32,26 @@ export const usePremiumContentAccess = (track: Nullable<Partial<Track>>) => {
     const isSignatureToBeFetched =
       isCollectibleGated &&
       !!trackId &&
-      premiumTrackSignatureMap[trackId] === undefined
+      premiumTrackSignatureMap[trackId] === undefined &&
+      !!user // We're only fetching a sig if the user is logged in
```

Comment:
This is the only real change, the rest is pretty much just imports and autoformatting by prettier